net_connection/src/json_protocol.rs

```diff
@@ -206,7 +214,9 @@ pub struct DhtMetaData {
     pub entry_address: Address,
 
     pub attribute: String,
-    pub content: serde_json::Value,
+    // single string or list of hashs
+    #[serde(rename = "contentList")]
```

Comment:
I don't know why you choose this formatting but this is the only place in the project where we choose camelCase.

Reply:
camelCase is far more standard as a convention for json properties. Changing from this at this point would be a much larger discussion, as all the networking modules, including n3h have been built to this protocol spec with camelCase.